blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClient.scala

```diff
@@ -13,18 +15,11 @@ import scalaz.{-\/, \/-}
 object BlazeClient {
   private[this] val logger = getLogger
 
-  def apply[A <: BlazeConnection](manager: ConnectionManager[A], config: BlazeClientConfig): Client = {
-
-    val shutdownTask = manager.shutdown().flatMap(_ => Task.delay {
-      // shutdown executor services that have been implicitly created for us
-      config.executor match {
-        case es: DefaultExecutorService =>
-          logger.info(s"Shutting down default ExecutorService: $es")
-          es.shutdown()
+  def apply[A <: BlazeConnection](manager: ConnectionManager[A],
```

Comment:
This is shutting down a connection manager it didn't create, and an opaque task that it didn't create.  That violates the "you opened it, you close it" mantra.  Should this be a private interface?